core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -114,11 +114,16 @@ protected function addHeadlineToTemplate(Template $template, $headline): void
     /**
      * @param string|array $cssID
      */
-    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null): void
+    protected function addCssAttributesToTemplate(Template $template, string $templateName, $cssID, array $classes = null, string $overrideCssID = null): void
```

Comment:
Are we allowed to add additional (nullable) properties to this method, or would that be a BC break if someone overrides it? The we might need to go with Symfony's `..., /** string $overrideCssID */` and `func_get_arg()`?

Reply:
A BC break only in PHP 8: https://3v4l.org/jAen0

So, a breaking behavior in the "real world" is unlikely IMHO.

Anyway, this method could have been `private`, no?

Reply:
It needs to be a `protected` method because you are supposed to be able to use it yourself if you implement your own `__invoke` function for example.

Reply:
Since this is an abstract class, which is intended to be extended, we must not change the method signature in a minor or patch version. We have to go with Symfony's `..., /** string $overrideCssID */` and `func_get_arg()`.